pixi/src/ui/recommendations/RecommendationsView.js

```diff
@@ -182,4 +197,9 @@ export default class RecommendationsView {
       recommendation.hidden = !commonValues.length;
     }
   }
+
+  onClickNext(recommendation) {
+    recommendation.classList.remove('expanded');
+    recommendation.nextSibling.classList.add('expanded');
```

Comment:
We also want to set `aria-expanded` to `"true"` (note it's a string).

By the way, the button is not available when there is not a `nextSibling` correct?

Reply:
Correct :)